src/components/checkboxFilter/index.tsx

```diff
@@ -76,7 +76,7 @@ function CheckboxFilter<ItemKey extends string>({
                       <HighlightedText
                         text={item.label}
                         highlightIndices={item.highlightIndices}
-                        wordBreak="break-all"
+                        overflowWrap="anywhere"
```

Comment:
as discussed in slack, I am fine with going with the deprecated value since we already have a tech task to tackle it in all places (since browsersupport isn't ideal for this one)